scripts/alfresco.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+
+sudo yum install -y git
```

Comment:
Why is git being installed into this app server?

Reply:
Required for galaxy, unless I misunderstand here